app/forms/citizens/percentage_home_form.rb

```diff
@@ -0,0 +1,12 @@
+module Citizens
+  class PercentageHomeForm
```

Comment:
Other forms are in the module `Applicants` and use `form_for :applicant`. To follow the same pattern, I think this form should be in the module `LegalAidApplications` (also sub-folder `legal_aid_applications`).

Reply:
good point